### PR TITLE
Adds keynote post for Matt Mitchell

### PR DESCRIPTION
**Pull request by rebkin05**
Changes proposed in this PR:

- Add blog post announcing Matt Mitchell

### CODE REVIEW

File: _posts/2019-08-14-matt-mitchell.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+author: Rebecca Kindschi
+category: General
+date: 2019-08-14 06:00:00
+layout: post
+image: /static/img/social/presenters/matt-mitchell.png
+title: "Announcing Our Keynotes: Matt Mitchell"
+---
+
+We're excited to announce our final; keynote speaker for DjangoCon US 2019: Matt Mitchell!
```
Comment (SaraDGore) [inline _posts/2019-08-14-matt-mitchell.md]: Is this colon supposed to be here?

File: _posts/2019-08-14-matt-mitchell.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+<img src="/static/img/speakers/matt-mitchell.jpg" alt="Photo of Matt Mitchell" style="width:45%; display:block; float:right;" />
+
+Matt Mitchell is a hacker and the Director of Digital Safety & Privacy at Tactical Tech (also known as the Tactical Technology Collective). In his work there Matt leads security training efforts, curricula, and organizational security for the organization in their mission to raise awareness about privacy, provide tools for digital security, and mobilize people to turn information into action.
+
+Matt is a well known security researcher, operational security trainer, and data journalist who founded & leads [CryptoHarlem](https://twitter.com/cryptoHarlem), impromptu workshops teaching basic cryptography tools to the predominately African American community in upper Manhattan.
+
+Matt’s talk is called Building Community and Supporting Others. He will share his experiences supporting the growth of others, including traditionally underrepresented groups in tech, inside and outside the Django community. He will draw parallels to Django's template architecture.
+
+Don't miss Jessica's keynote! [Buy your ticket]({{site.ticket_link}}) today.
```
Comment (SaraDGore) [inline _posts/2019-08-14-matt-mitchell.md]: you have Jessica's name here

File: _posts/2019-08-14-matt-mitchell.md (hunk 1 above)
Comment (rebkin05) [reply _posts/2019-08-14-matt-mitchell.md]: Nope, I'll remove. Good catch, thank you!

File: _posts/2019-08-14-matt-mitchell.md (hunk 2 above)
Comment (rebkin05) [reply _posts/2019-08-14-matt-mitchell.md]: You're a 💎

**Comment by rebkin05** [comment]
thanks, @SaraDGore!

**Pull request merged**